### PR TITLE
Add release info for sui-full-node doc

### PR DESCRIPTION
**Pull request by ebmifa**
## Description 
Add release info for sui-full-node doc

## Test plan 
@ronny-mysten !

### CODE REVIEW

**Review by johnjmartin — APPROVED** [review]

File: docs/content/guides/operator/updates.mdx [hunk 1]
```diff
@@ -3,13 +3,27 @@ title: Updating a Full Node
 description: Update your Sui Full node to the latest version to remain in sync with the network.
 ---
 
-## Release process
+## Sui Release process
```
Comment (ronny-mysten) [inline docs/content/guides/operator/updates.mdx:6]:
```suggestion
## Sui release process

Each Sui branch is released on a consistent schedule. There are extenuating circumstances that might delay releases occasionally, but these delays are rare and communicated through [official channels](#communication).  
```

File: docs/content/guides/operator/updates.mdx [hunk 2]
```diff
@@ -3,13 +3,27 @@ title: Updating a Full Node
...
+
+`devnet` → every week on Mondays\
```
Comment (ronny-mysten) [inline docs/content/guides/operator/updates.mdx:8]:
```suggestion
- `devnet`: every week on Mondays.
```

File: docs/content/guides/operator/updates.mdx [hunk 3]
```diff
@@ -3,13 +3,27 @@ title: Updating a Full Node
...
+`testnet` → every week on Tuesdays\
```
Comment (ronny-mysten) [inline docs/content/guides/operator/updates.mdx:9]:
```suggestion
- `testnet`: every week on Tuesdays.
```

File: docs/content/guides/operator/updates.mdx [hunk 4]
```diff
@@ -3,13 +3,27 @@ title: Updating a Full Node
...
+`mainnet` → every two weeks on Wednesdays
```
Comment (ronny-mysten) [inline docs/content/guides/operator/updates.mdx:10]:
```suggestion
- `mainnet`: every two weeks on Wednesdays.
```

File: docs/content/guides/operator/updates.mdx [hunk 5]
```diff
@@ -3,13 +3,27 @@ title: Updating a Full Node
...
+
+Note: For more details, please see each network's release schedule and configuration [here](https://sui.io/networkinfo).
```
Comment (ronny-mysten) [inline docs/content/guides/operator/updates.mdx:12]:
```suggestion
:::info
For additional details, see each network's [release schedule and configuration](https://sui.io/networkinfo).
:::
```

File: docs/content/guides/operator/updates.mdx [hunk 6]
```diff
@@ -3,13 +3,27 @@ title: Updating a Full Node
...
 
 Whenever Sui releases a new version, you must update your Full node with the release to ensure compatibility with the network it connects to. For example, if you use Sui Testnet you should install the version of Sui running on Sui Testnet.
 
 Any release that contains a protocol change will need to be followed before the protocol upgrade takes place (when enough stake within the validator set upgrades, the new protocol version is enacted in the next epoch).
 If you do not update your Full node, you will not be able to connect to the network after the protocol upgrade takes place.
 
+## Communication
+
+Releases get announced on [Sui Discord server](https://discord.com/invite/sui) and [node-operators](https://groups.google.com/a/groups.sui.io/g/node-operators) Google group\
```
Comment (ronny-mysten) [inline docs/content/guides/operator/updates.mdx:21]:
```suggestion
Releases are announced on [Sui Discord server](https://discord.com/invite/sui) and [node-operators](https://groups.google.com/a/groups.sui.io/g/node-operators) Google group.
```

File: docs/content/guides/operator/updates.mdx [hunk 7]
```diff
@@ -3,13 +3,27 @@ title: Updating a Full Node
...
+Discord channels:\
```
Comment (ronny-mysten) [inline docs/content/guides/operator/updates.mdx:22]:
```suggestion
Discord channels:
```

File: docs/content/guides/operator/updates.mdx [hunk 8]
```diff
@@ -3,13 +3,27 @@ title: Updating a Full Node
...
+`devnet` → `#devnet-updates` channel\
```
Comment (ronny-mysten) [inline docs/content/guides/operator/updates.mdx:23]:
```suggestion
- `devnet`: [`#devnet-updates` channel](https://discord.com/channels/916379725201563759/1004638487078772736)
```

File: docs/content/guides/operator/updates.mdx [hunk 9]
```diff
@@ -3,13 +3,27 @@ title: Updating a Full Node
...
+`testnet` → `#tn-validator-announcements`, `#testnet-updates` ⁠and `#node-announcements` channels\
```
Comment (ronny-mysten) [inline docs/content/guides/operator/updates.mdx:24]:
```suggestion
- `testnet`: [`#tn-validator-announcements`](https://discord.com/channels/916379725201563759/1003660994381353101), [`#testnet-updates`](https://discord.com/channels/916379725201563759/1095151359642304612), ⁠and [`#node-announcements`](https://discord.com/channels/916379725201563759/1002231298888306718) channels.
```

File: docs/content/guides/operator/updates.mdx [hunk 10]
```diff
@@ -3,13 +3,27 @@ title: Updating a Full Node
...
+`mainnet` → `⁠#mn-validator-announcements`, `#mainnet-updates` and `#node-announcements` channels
```
Comment (ronny-mysten) [inline docs/content/guides/operator/updates.mdx:25]:
```suggestion
- `mainnet`: [`⁠#mn-validator-announcements`](https://discord.com/channels/916379725201563759/1093852827627040768), [`#mainnet-updates`](https://discord.com/channels/916379725201563759/1103082453792464906), and [`#node-announcements`](https://discord.com/channels/916379725201563759/1002231298888306718) channels.
```

File: docs/content/guides/operator/updates.mdx (hunk 1 above)
Comment (ronny-mysten) [reply docs/content/guides/operator/updates.mdx:6]: You might have a better way to state this.

**Review by ronny-mysten — APPROVED** [review]
Thanks @ebmifa !

File: docs/content/guides/operator/updates.mdx [hunk 11]
```diff
@@ -3,13 +3,28 @@ title: Updating a Full Node
 description: Update your Sui Full node to the latest version to remain in sync with the network.
 ---
 
-## Release process
+## Sui release process
+
+- `devnet`: deployed every week on Mondays
```
Comment (ronny-mysten) [inline docs/content/guides/operator/updates.mdx:8]:
```suggestion
- `devnet`: Deployed every week on Mondays.
```

File: docs/content/guides/operator/updates.mdx [hunk 12]
```diff
@@ -3,13 +3,28 @@ title: Updating a Full Node
...
+- `testnet`: deployed every week on Tuesdays
```
Comment (ronny-mysten) [inline docs/content/guides/operator/updates.mdx:9]:
```suggestion
- `testnet`: Deployed every week on Tuesdays.
```

File: docs/content/guides/operator/updates.mdx [hunk 13]
```diff
@@ -3,13 +3,28 @@ title: Updating a Full Node
...
+- `mainnet`: deployed every two weeks on Wednesdays
```
Comment (ronny-mysten) [inline docs/content/guides/operator/updates.mdx:10]:
```suggestion
- `mainnet`: Deployed every two weeks on Wednesdays.
```

File: docs/content/guides/operator/updates.mdx [hunk 14]
```diff
@@ -3,13 +3,28 @@ title: Updating a Full Node
...
+
+> **_Info:_** For additional details, see each network's [release schedule and configuration](https://sui.io/networkinfo).
```
Comment (ronny-mysten) [inline docs/content/guides/operator/updates.mdx:12]:
```suggestion
:::info
 For additional details, see each network's [release schedule and configuration](https://sui.io/networkinfo).
 :::
```

File: docs/content/guides/operator/updates.mdx [hunk 15]
```diff
@@ -3,13 +3,28 @@ title: Updating a Full Node
...
 
 Whenever Sui releases a new version, you must update your Full node with the release to ensure compatibility with the network it connects to. For example, if you use Sui Testnet you should install the version of Sui running on Sui Testnet.
 
 Any release that contains a protocol change will need to be followed before the protocol upgrade takes place (when enough stake within the validator set upgrades, the new protocol version is enacted in the next epoch).
 If you do not update your Full node, you will not be able to connect to the network after the protocol upgrade takes place.
 
+## Communication
+Each Sui network is deployed on a consistent schedule. There are extenuating circumstances that might delay releases occasionally, but these delays are rare and communicated through.
```
Comment (ronny-mysten) [inline docs/content/guides/operator/updates.mdx:20]:
```suggestion  
```

File: docs/content/guides/operator/updates.mdx [hunk 16]
```diff
@@ -3,13 +3,28 @@ title: Updating a Full Node
 description: Update your Sui Full node to the latest version to remain in sync with the network.
 ---
 
-## Release process
+## Sui release process
+
```
Comment (ronny-mysten) [inline docs/content/guides/operator/updates.mdx:7]:
```suggestion

Each Sui network is deployed on a consistent schedule. There are extenuating circumstances that might delay releases occasionally, but these delays are rare and communicated through [official channels](#communication).

```

File: docs/content/guides/operator/updates.mdx [hunk 17]
```diff
@@ -3,13 +3,28 @@ title: Updating a Full Node
...
 Whenever Sui releases a new version, you must update your Full node with the release to ensure compatibility with the network it connects to. For example, if you use Sui Testnet you should install the version of Sui running on Sui Testnet.
 
 Any release that contains a protocol change will need to be followed before the protocol upgrade takes place (when enough stake within the validator set upgrades, the new protocol version is enacted in the next epoch).
 If you do not update your Full node, you will not be able to connect to the network after the protocol upgrade takes place.
 
+## Communication
+Each Sui network is deployed on a consistent schedule. There are extenuating circumstances that might delay releases occasionally, but these delays are rare and communicated through.  
+
+Releases are announced on [Sui Discord server](https://discord.com/invite/sui) and [node-operators](https://groups.google.com/a/groups.sui.io/g/node-operators) Google group.
+##### Discord channels:
```
Comment (ronny-mysten) [inline docs/content/guides/operator/updates.mdx:23]:
```suggestion

### Discord channels
```

**Review by ronny-mysten — APPROVED** [review]

File: docs/content/guides/operator/updates.mdx [hunk 18]
```diff
@@ -3,13 +3,32 @@ title: Updating a Full Node
...
+
+Each Sui network is deployed on a consistent schedule. There are extenuating circumstances that might delay releases occasionally, but these delays are rare and communicated through [official channels](#communication).
+
+- `devnet`: Deployed every week on Mondays.
+- `testnet`: Deployed every week on Tuesdays.
+- `mainnet`: Deployed every two weeks on Wednesdays.
+
+:::info
+ For additional details, see each network's [release schedule and configuration](https://sui.io/networkinfo).
+ :::
```
Comment (ebmifa) [inline docs/content/guides/operator/updates.mdx:16]: @ronny-mysten this does not render correctly
Screenshot 2024-10-15 at 3 22 10 PM

**Pull request merged**